modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-bare-metal-config-yaml-{context}"]
+= Sample `install-config.yaml` file for bare metal
+
+You can customize the `install-config.yaml` file to specify more details about
+your {product-title} cluster's platform or modify the values of the required
+parameters.
+
+[source,yaml]
+----
+apiVersion: v1beta3
+baseDomain: example.com <1>
+compute:
+- name: worker
+  replicas: 1
+controlPlane:
+  name: master
+  replicas: 1
+metadata:
+  name: test <2>
+networking:
```

Comment:
@abhinavdahiya, do you have opinions about what we should say about networking in the bare metal context?

Reply:
`clusterNetworks`  and `serverCIDR` are same as installer created aws etc. platform. so we can re-use definition from there.?